translations/nl.ts

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="nl">
+<context>
+    <name>PagePower</name>
+    <message>
+        <source>Suspend</source>
+        <translation>Pauzestand</translation>
```

Comment:
I think "sluimerstand" could be more fitting.

Reply:
I've never heard that term before. “Pauzestand” is the regular term, at least on Linux. So I'd like to keep it consistent.